src/components/GradeBookContentContainer.tsx

```diff
@@ -7,112 +7,113 @@ import GradeItemAccordion from './GradeItemAccordion';
 const ContentContainer = styled.div`
 	color: ${(props) => props.theme.colors.text[1]};
 	font-weight: bold;
-`;
+	padding: 50px 0px 50px 50px;
 
-const TitleRow = styled.div`
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	width: 100%;
-	border-style: hidden hidden solid hidden;
-	color: ${(props) => props.theme.colors.gray[1]};
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 75px 75px;
+	grid-template-areas:
+		'columnItemHeader columnItemHeader columnItemHeader columnItemHeader buttonColumn'
+		'courseTitleSpace courseItemTitle courseWeightTitle courseGradeTitle courseDuedateTitle'
+		'gradeItemAccordian gradeItemAccordian gradeItemAccordian gradeItemAccordian gradeItemAccordian';
 `;
 
 const ColumnItemHeader = styled.div`
-	display: flex;
-	flex-direction: column;
-	text-align: left;
-	flex-basis: 100%;
-	flex: 1;
+	grid-area: columnItemHeader;
 	color: ${(props) => props.theme.colors.text[1]};
-`;
 
-const ButtonColumn = styled.div`
-	display: flex;
-	flex-direction: column;
-	flex-basis: initial;
-	margin: 1.5em;
-	width: 15em;
+	border: 1px solid #828282;
```

Comment:
Another super minor comment, but could this be changed to use the theme, too? :)